Userland/Applications/FileManager/PropertiesWindowPDFTab.gml

```diff
@@ -0,0 +1,195 @@
+                spacing: 12
+            }
+
+            @GUI::Label {
+                text: "Version:"
+                text_alignment: "TopLeft"
+                fixed_width: 80
+            }
+
+            @GUI::Label {
+                name: "pdf_version"
+                text: "1.2"
+                text_alignment: "TopLeft"
+            }
+        }
+
+        @GUI::Widget {
+            layout: @GUI::HorizontalBoxLayout {
+                spacing: 12
+            }
+
+            @GUI::Label {
+                text: "Page count:"
+                text_alignment: "TopLeft"
+                fixed_width: 80
+            }
+
+            @GUI::Label {
+                name: "pdf_page_count"
+                text: "42"
+                text_alignment: "TopLeft"
+            }
+        }
+
+        @GUI::Widget {
+            layout: @GUI::HorizontalBoxLayout {
+                spacing: 12
+            }
+
+            @GUI::Label {
+                text: "Title:"
+                text_alignment: "TopLeft"
+                fixed_width: 80
+            }
+
+            @GUI::Label {
+                name: "pdf_title"
+                text: "How to Shave Your Yak"
+                text_alignment: "TopLeft"
+            }
+        }
+
+        @GUI::Widget {
+            layout: @GUI::HorizontalBoxLayout {
+                spacing: 12
+            }
+
+            @GUI::Label {
+                text: "Author:"
+                text_alignment: "TopLeft"
+                fixed_width: 80
+            }
+
+            @GUI::Label {
+                name: "pdf_author"
+                text: "Andreas Kling"
+                text_alignment: "TopLeft"
+            }
+        }
+
+        @GUI::Widget {
+            layout: @GUI::HorizontalBoxLayout {
+                spacing: 12
+            }
+
+            @GUI::Label {
+                text: "Subject:"
+                text_alignment: "TopLeft"
+                fixed_width: 80
+            }
+
+            @GUI::Label {
+                name: "pdf_subject"
+                text: "Hair control methods for these beasts"
+                text_alignment: "TopLeft"
+            }
+        }
+
+        @GUI::Widget {
+            layout: @GUI::HorizontalBoxLayout {
+                spacing: 12
+            }
+
+            @GUI::Label {
+                text: "Keywords:"
+                text_alignment: "TopLeft"
+                fixed_width: 80
+            }
+
+            @GUI::Label {
+                name: "pdf_keywords"
+                text: "yak, shaving"
+                text_alignment: "TopLeft"
+            }
+        }
+
+        @GUI::Widget {
+            layout: @GUI::HorizontalBoxLayout {
+                spacing: 12
+            }
+
+            @GUI::Label {
+                text: "Creator:"
+                text_alignment: "TopLeft"
+                fixed_width: 80
+            }
+
+            @GUI::Label {
+                name: "pdf_creator"
+                text: "SerenityOS Office"
```

Comment:
Is there a benefit to having these non-real default text values? Seems like they could just be empty, then you don't have to go through to effort of setting them to an empty string if an error occurs

Reply:
Beyond making it a little nicer to work with in GML Playground, and my own amusement, not really. :sweat_smile: